examples/basic-examples/assets/ripple-icon/custom_icon_lock_open.js

```diff
@@ -0,0 +1,13 @@
+import Icon from '../../../../packages/components/Atoms/Icon/Icon.vue'
```

Comment:
I'll replace this with `@dpc-sdp/ripple-icon/Icon.vue` once this is good to merge. This is only for this branch to work.